src/dcommands/remind.js

```diff
@@ -51,22 +51,20 @@ Example: ${example}`);
             starttime: r.epochTime(moment().unix()),
             endtime: r.epochTime(moment().add(duration).unix())
         });
-        await bu.send(msg, `:alarm_clock: Ok! I'll remind you ${channel ? 'here' : 'in a DM'} ${duration.humanize(true)}! :alarm_clock: `);
+        await bu.send(msg, `:alarm_clock: Ok! I'll remind you ${channel ? 'here' : 'in a DM'} <t:${moment().add(duration).unix()}:R>! :alarm_clock: `);
```

Comment:
Can you externalize the moment calculation out of the string interpolation if possible? Look above at line 52, it's the same value used over there 😄

Reply:
I commented the same thing 🤔 weird

Reply:
Aight